README.md

```diff
@@ -1,34 +1,38 @@
 ArcGIS Runtime SDK for iOS Samples [![](https://user-images.githubusercontent.com/2257493/54144188-6fe0fc00-43e8-11e9-8cf5-229af80f604a.png)](https://itunes.apple.com/us/app/arcgis-runtime-sdk-for-ios/id1180714771)
 ==========================
 
-This repository contains Swift sample code demonstrating the capabilities of [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/) and how to use them in your own app. The project that can be opened in Xcode and run on a simulator or a device. Or you can [download the app from the App Store](https://itunes.apple.com/us/app/arcgis-runtime-sdk-for-ios/id1180714771) on your iOS device.
+This repository contains Swift sample code demonstrating the capabilities of [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/latest/) and how to use them in your own app. The project that can be opened in Xcode and run on a simulator or a device. Or you can [download the app from the App Store](https://itunes.apple.com/us/app/arcgis-runtime-sdk-for-ios/id1180714771) on your iOS device.
 
 ![](SampleApp.png)
 
 The ```master``` branch of this repository contains samples configured for the latest available version of ArcGIS Runtime SDK for iOS. For samples configured for older versions of the SDK,  look under the ```Releases``` tab for a specific version.
 
-
 ## Features
 
 * Maps - Open, create, inteact with and save maps
 * Layers - Layer types offered by the SDK
-* Features - Working with Feature layers
-* Edit Data - Adding, deleting and editing features
+* Features - Working with feature layers
 * Display Information - Displaying graphics, popups and callouts
 * Search - Finding an address
+* Edit Data - Adding, deleting and editing features
 * Geometry - Displaying geometries
 * Route & Directions - Find a route around barriers and get turn-by-turn directions
+* Analysis - 
+* Cloud & Portal - 
 * Scenes - Display scenes, 3D symbols, and scene layers
+* Utility Network - 
+* Augmented Reality - 
```

Comment:
I've re-ordered the categories according to the samples app. However, we need to come up with a few one-line description for each new category.